charts/adot-exporter-for-eks-on-ec2/values.schema.json

```
@@ -1088,7 +1088,7 @@
                                 },
                                 "repository": {
                                     "type": "string",
-                                    "pattern": "(^$|^[/a-z0-9-_]+)$"
+                                    "pattern": "^[a-z0-9\\.\\-_\\/]+$"
```

```suggestion
                                    "pattern": "^[a-z0-9\\.\\-_\\/]*$"
```

The empty string was previously valid here.

---

OK, fixed